src/wmnds/components/phase-indicator/_phase-indicator-link.njk

```diff
@@ -0,0 +1,17 @@
+{% macro wmndsPhaseIndicatorLink(params) %}
+<div class="wmnds-grid wmnds-banner-container">
+  <div class="wmnds-col-auto wmnds-float-left wmnds-m-r-xsm">
+     <a href="/" target="_self" class="wmnds-phase-indicator" title="Beta homepage of West Midlands Network Design System">
+       Beta
+     </a>
```

Comment:
Like below, can you import the phase indicator as a component.

Reply:
implemented